ZukoAzula2016/src/org/team1540/zukoazula/Portcullis.java

```diff
@@ -21,22 +21,28 @@
     private static final FloatIO leftEncoder = leftGrabMotor.modEncoder().getEncoderPosition();
     private static final FloatIO rightEncoder = rightGrabMotor.modEncoder().getEncoderPosition();
 
-    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f);
+    private static final FloatInput control = ZukoAzula.controlBinding.addFloat("Portcullis Grabber Axis").deadzone(0.3f).negated();
```

Comment:
You negated the axis and you didn't reverse the motors. So do the motors move the opposite way now?